src/data_access/api/SpotifyAPI.java

```diff
@@ -29,50 +32,50 @@ public SpotifyAPI(SongFactory songFactory){
     public Song getRandomSongFromGenre(String genre) {
         JSONObject responseBody = retrieveSongData(genre);
         JSONArray songsArray = responseBody.getJSONObject("tracks").getJSONArray("items");
+
         JSONObject item;
         Song song = null;
+
         int i = 0;
-        do{
+        do {
```

Comment:
Really small thing, but just make sure the spacing is consistent between 'do' and '{'

Reply:
Hmm, the current code commit shows that there is a space already. Can you double-check on your copy that the space isn't there?